docs/INSTALLATION.md

```diff
@@ -0,0 +1,398 @@
+* 4. [Development](#Development)
+* 5. [End-to-End API tests](#End-to-EndAPItests)
+* 6. [CI/CD and Test Automation](#CICDandTestAutomation)
+	* 6.1. [Github Actions](#GithubActions)
+	* 6.2. [Test Github Action workflows locally](#TestGithubActionworkflowslocally)
+
+<!-- vscode-markdown-toc-config
+	numbering=true
+	autoSave=true
+	/vscode-markdown-toc-config -->
+<!-- /vscode-markdown-toc -->
+
+> **_New Developers:_** Consult the [development guide](./DEVELOPMENT.md) for development flow and guidance
+
+##  1. <a name='Prerequisites'></a>Prerequisites
+
+###  1.1. <a name='UnderstandingGoogleCloud'></a>Understanding Google Cloud
+
+We recommend the following resources to get familiar with Google Cloud and microservices.
+
+- What is Microservice Architecture: https://cloud.google.com/learn/what-is-microservices-architecture
+- Kubernetes:
+  - Learn about the basics of Kubernetes: https://kubernetes.io/docs/concepts/overview/
+  - Google Kubernetes Engine (GKE) overview: https://cloud.google.com/kubernetes-engine/docs/concepts/kubernetes-engine-overview
+  - Skaffold, a command line tool that facilitates continuous development for container based & Kubernetes applications: https://skaffold.dev/docs/
+- CloudRun:
+  - Serverless container deployment and execution with CloudRun: https://cloud.google.com/run/docs/overview/what-is-cloud-run
+
+###  1.2. <a name='Toolrequirements:'></a>Tool requirements:
+
+| Tool  | Required Version  | Documentation site |
+|---|---|---|
+| gcloud CLI          | Latest     | https://cloud.google.com/sdk/docs/install |
+| Terraform           | >= v1.3.7  | https://developer.hashicorp.com/terraform/downloads |
+| Skaffold (for GKE)  | >= v2.0.4  | https://skaffold.dev/ |
+| Kustomize (for GKE) | >= v4.3.1  | https://kustomize.io/ |
+| Cookiecutter        | >=2.1.1    | https://cookiecutter.readthedocs.io/en/latest/installation.html#install-cookiecutter |
+
+###  1.3. <a name='RequiredpackagesfordeployingtoGKEcluster:'></a>Required packages for deploying to GKE cluster:
+> You can skip this section if you choose to deploy microservices to CloudRun only.
+
+Install **skaffold (2.0.4 or later)** and kustomize:
+
+- For Google CloudShell or Linux/Ubuntu:
+  ```
+  export SKAFFOLD_VERSION=v2.0.4
+  curl -Lo skaffold https://storage.googleapis.com/skaffold/releases/$SKAFFOLD_VERSION/skaffold-linux-amd64 && \
+  sudo install skaffold /usr/local/bin/
+  ```
+- For MacOS:
+  ```
+  brew install skaffold kustomize
+  ```
+- For Windows:
+  ```
+  choco install -y skaffold kustomize gcloudsdk
+  ```
+
+###  1.4. <a name='AppleM1ChipSupportforTerraformOptional'></a>Apple M1 Chip Support for Terraform (Optional)
+
+If you are running commands on an Apple M1 chip Macbook, make sure run the following to add M1 support for Terraform:
+```
+brew install kreuzwerker/taps/m1-terraform-provider-helper
+m1-terraform-provider-helper activate
+m1-terraform-provider-helper install hashicorp/template -v v2.2.0
+```
```

Comment:
The info above is already mentioned in TROUBLESHOOTING.md

Reply:
It's intentional, when users walk through the installation, these are the minimum steps they'll need to run when using Mac.